BF-it.py

```diff
@@ -53,5 +53,5 @@ def compile_file(input_file, output_file, run, minify_file):
 
 if __name__ == '__main__':
     input_file, output_file, run_file, minify_file = process_args()
-    #input_file = "examples/games/tic_tac_toe.code"
+    # input_file = "examples/games/tic_tac_toe.code"
```

Comment:
Please remove this line completely, it was part of debug and pushed by mistake :)